game-service/src/test/java/com/lms/gameservice/PaymentServiceClientTest.java

```diff
@@ -1,4 +1,4 @@
-package com.lms.gameservice.service;

```

Comment:
Whats this class purpose?

Reply:
Callum made this, I think it is to pay when joining a game. I'll have a look cause there is no updates for the prize pot when people join and I don't think there is a check to see if a person has the balance to join.